code/game/jobs/job/assistant.dm

```diff
@@ -10,7 +10,7 @@
 	selection_color = "#dddddd"
 	access = list()			//See /datum/job/assistant/get_access()
 	minimal_access = list()	//See /datum/job/assistant/get_access()
-	alt_titles = list("Technical Assistant","Medical Intern","Research Assistant","Visitor")
+	alt_titles = list("Technical Assistant","Medical Intern","Research Assistant","Security Cadet", "Visitor")

```

Comment:
Possibly a mistake on my part. This would add them as an alternate title to Assistant, which they're not really meant to be if they have their own job datums.